app/Http/Controllers/Store/CentreController.php

```diff
@@ -98,8 +111,9 @@ public function exportRegistrationsSummary()
             $row = [
                 // TODO: null objects when DB is duff: try/catch findOrFail() in the relationship?
                 "RVID" => ($reg->family) ? $reg->family->rvid : 'Family not found',
+                "Area" => ($reg->centre->sponsor) ? $reg->centre->sponsor->name : 'Area not found',
                 "Centre" => ($reg->centre) ? $reg->centre->name : 'Centre not found',
-                "Primary Carer" => ($reg->family->carers->first()) ? $reg->family->carers->first()->name : null,
+                "Primary Carer" => ($reg->family->pri_carer) ? $reg->family->pri_carer : null,
```

Comment:
Do you need to check `$reg->family` first to avoid blow up?

Reply:
Have discovered null coalesce `(thing->otherThing->property) ?? 'can't find it'` does not throw a fit on failing. Will switch to that.